### ISSUE TITLE
Legacy People API

### ISSUE THREAD

**Issue by EagleOne33** [zquestz/omniauth-google-oauth2#387]
Hello,

I get the following error: 

```
"error": {
    "code": 403,
    "message": "Legacy People API has not been used in project xxxxxxxxxx before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/legacypeople.googleapis.com/overview?project=xxxxxxxxxx then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.",
    "status": "PERMISSION_DENIED",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.Help",
        "links": [
          {
            "description": "Google developers console API activation",
            "url": "https://console.developers.google.com/apis/api/legacypeople.googleapis.com/overview?project= xxxxxxxxxx"
          }
        ]
      }
    ]
```

I tried to go to the link of course but it seems that for new apps, legacypeople API is retired and replaced by Google People API. 
So I activated Google People API but it doesn't change anything... Any help?

**Comment by EagleOne33** [comment]
Sorry I was using an old version. From 0.8.0 it's working

**Comment by dorianmariefr** [comment]
thanls, doing `bundle update omniauth-google-oauth2` fixed it, not sure why I was on 0.5.4

**Comment by ssaunier** [comment]
Maybe you had an old version of `google_drive` in your `Gemfile` which locked the `omniauth-google-oauth2` to `0.5.4`. That's what I had.
